webapp/channels/src/components/add_user_to_group_multiselect/add_user_to_group_multiselect.tsx

```diff
@@ -245,6 +245,12 @@ export class AddUserToGroupMultiSelect extends React.PureComponent<Props, State>
                 backButtonText={this.props.backButtonText}
                 maxValues={maxValues}
                 numRemainingText={numRemainingText}
+                noteText={!this.state.values.length && (
```

Comment:
The UX for this noteText doesn't seem to align with the other fields here of required: 

<img width="617" alt="Screenshot 2025-05-23 at 11 25 55 AM" src="https://github.com/user-attachments/assets/8a8c84cb-d0dc-49bd-9cee-f4ce25bd676c" />

@asaadmahmood thoughts on this? Can we adjust that styling overall?

Reply:
FWIW that's the default UX on that `Multiselect` component, for some reason.

Reply:
Looks much better 🎉 

<img width="667" alt="Screenshot 2025-05-28 at 6 55 33 PM" src="https://github.com/user-attachments/assets/546d2418-61a6-4823-a2a8-a6c50fbcb415" />